tests/test_func.py

```diff
@@ -1,10 +1,22 @@
 import unittest
+from unittest.mock import patch
 from funcs import func
 
 class Test(unittest.TestCase):
 
     def test_sum(self):
         self.assertEqual(func.sum(2,3), 5)
 
+    # Any misformatted functions would not pass flake8 format checks. All related tests would not be executed.
```

Comment:
Remove this comment when running them as separate pipelines.

Reply:
Removed.